asusd/src/config.rs

```diff
@@ -90,13 +90,13 @@ impl StdConfig for Config {
         }
     }
 
-    fn config_dir() -> std::path::PathBuf {
-        std::path::PathBuf::from(crate::CONFIG_PATH_BASE)
-    }
-
     fn file_name(&self) -> String {
         CONFIG_FILE.to_owned()
     }
+
```

Comment:
Please don't do spurious changes like this, it adds noise to the meat of the PR.